content/source/docs/enterprise/api/ssh-keys.html.md

```diff
@@ -96,26 +109,45 @@ curl \
 
 ## Create an SSH Key
 
-Create an SSH Key.
+`POST /organizations/:organization_name/ssh-keys`
+
+Parameter            | Description
+---------------------|------------
+`:organization_name` | The name of the organization to create an SSH key in. The organization must already exist, and the token authenticating the API request must belong to the "owners" team or a member of the "owners" team.
+
+-> **Note:** This endpoint is disabled for [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) or [team token](../users-teams-organizations/service-accounts.html#team-service-accounts).
+
+Status  | Response                                   | Reason
+--------|--------------------------------------------|-------
+[201][] | [JSON API document][] (`type: "ssh-keys"`) | Success
+[422][] | [JSON API error object][]                  | Validation errors
```

Comment:
I'm unsure that "Validation errors" is very helpful for 422. Is that standard per the template?

Reply:
That might've been me. I think something more descriptive to the validation failure at hand rather than anything to do with the HTTP side of things would help people understand the failure more.